readme.md

```diff
@@ -24,13 +24,22 @@ You've written the following HTML. When you look at it in your browser, it's jus
 </html>
 ```
 
-> Your answer...
+```
+ANSWER:
+
+The title tag is not closed! It should be:
+  </title>
+```
 
 ## Question 2
 
 What's the purpose of the `alt` attribute on image tags?
 
-> Your answer...
+```
+ANSWER:
+
+Alt provides an alternative text that describes an image if the image cannot be displayed on the browser for whatever reason, or if the user utilizes a screen reader.
+```
```

Comment:
:+1: 

(As a note, it's also useful for SEO for image search tools... helps them guess what a given image is about)